public/data/kapost_v1.json

```diff
@@ -1,6 +1,305 @@
 {
   "endpoints":[
     {
+      "name":"Ideas <pre>/api/v1/ideas/</pre>",
+      "methods":[
+        {
+          "MethodName": "List Ideas",
+          "Synopsis": "Retrieve a filtered, paged list of idea resources",
+          "HTTPMethod": "GET",
+          "URI": "/ideas",
+          "RequiresOAuth": "N",
+          "parameters":[
+            {
+              "Name":"detail",
+              "Required":"N",
+              "Default":"basic",
+              "Type":"enumerated",
+              "EnumeratedList":[
+                "basic",
+                "full"
+              ],
+              "EnumeratedDescription":{
+                "basic": "A short, summary response",
+                "full": "All idea details are returned"
+              },
+              "Description":"The level of detail in the response for each record"
+            },
+            {
+              "Name":"page",
+              "Required":"N",
+              "Default":"1",
+              "Type":"integer",
+              "Description":"The (1-indexed) page number to retrieve."
+            },
+            {
+              "Name":"per_page",
+              "Required":"N",
+              "Default":"25",
+              "Type":"integer",
+              "Description":"The number of entries to retrieve per page."
+            },
+            {
+              "Name":"user_id",
```

Comment:
Even if this user_id param is real I think we should not expose it. It doesn't make sense to me.